### PR TITLE
Package "@action/artifact@v1" is getting deprecated on June 30, 2024

### LINKED ISSUE #63: Package "@actions/artifact" v1 will be deprecated in June 30, 2024
> Starting November 30, 2024, GitHub Actions customers will no longer be able to use v3 of [actions/upload-artifact](https://github.com/actions/upload-artifact) or [actions/download-artifact](https://github.com/actions/download-artifact). 

> The deprecation of v3 will be similar to the [previously announced v1 and v2 deprecation plans](https://github.blog/changelog/2024-02-13-deprecation-notice-v1-and-v2-of-the-artifact-actions/), which is scheduled to take place on June 30, 2024.

https://github.blog/changelog/2024-04-16-deprecation-notice-v3-of-the-artifact-actions/

### PR DESCRIPTION
**Pull request by alexkuc**
Fix #63

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,5 +47,8 @@
 		"prettier": "^2.1.2",
 		"ts-jest": "^29.0.3",
 		"typescript": "^4.0.5"
+	},
+	"resolutions": {
+		"strip-ansi": "6.0.1"
```
Comment (alexkuc) [inline package.json:52]: https://github.com/eslint/eslint/discussions/17215#discussioncomment-7884236

**Review by tn3rb — APPROVED** [review]
looks good

there's a bunch of auto generated codde review comments frrom githb about variables that are defined but not used. dunno if you want to do anything about that though

**Comment by alexkuc** [comment]
> there's a bunch of auto generated codde review comments frrom githb about variables that are defined but not used. dunno if you want to do anything about that though

Addressed

**Review by tn3rb — APPROVED** [review]
👍🏻

**Pull request merged**